packages/radio-button/src/__stories__/renderStory.js

```diff
@@ -1,21 +1,37 @@
 import React from "react";
+import { css } from "emotion";
+import Label from "@hig/label";
 import DefaultExport from "../index";
 import getKnobs from "./getKnobs";
 
 export default function renderStory(props) {
   const { children, ...otherProps } = getKnobs(props);
+  const outerStyle = {
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "center",
+    marginBottom: "5px"
+  };
 
   return (
     <form>
-      <DefaultExport {...otherProps}>{children}</DefaultExport>
-      <DefaultExport
-        {...otherProps}
-        defaultChecked={null}
-        label="Secondary"
-        value="Secondary"
-      >
-        {children}
-      </DefaultExport>
+      <div className={css(outerStyle)}>
+        <DefaultExport {...otherProps} id="radio-button-1">
+          {children}
+        </DefaultExport>
+        <Label htmlFor="radio-button-1">Radio Button 1</Label>
+      </div>
+      <div className={css(outerStyle)}>
+        <DefaultExport
+          {...otherProps}
+          checked
```

Comment:
i think `checked` here is causing some issues w/ storybook. w/ checked set to true it should be controlled but if i click the first radio button a few times it switches, `defaultChecked` might be better for just setting an initially checked radio button.

Reply:
might be my local build but i'm actually not seeing the theme data stuff in my local `storybook`

Reply:
I think passing along `checked` was a mistake, we should be relying on `defaultChecked` instead and that fixed the behavior in the storybook.